### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by tjmac21**

### CODE REVIEW

File: src/components/home/Header.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default class Header extends Component {
       };
     }
     fetchData(){
-        alert('Refresh');
+        alert('Refresh/Get New Data');
```
Comment (mharr-cmpt) [inline src/components/home/Header.js]: what this do?

Comment (mharr-cmpt) [reply src/components/home/Header.js]: or rather why this change?

File: src/components/home/MessageBox.js [hunk 2]
```diff
@@ -18,14 +19,28 @@ export default class MessageBox extends Component {
...
+      //figure out json structure of each "drop" object
+      // object needs:
+      //    unique id
+      //    user info( need to know who to attribute to) (can set to anon)
+      //    location (long/lat)
+      //    radius it will be visible 
+      //    message
+      //    img?
+      //    life span
+      //    store to firebase db
```
Comment (mharr-cmpt) [inline src/components/home/MessageBox.js:32]: this is just the object properties that you think we'll need, designed by you, right?  img could be int he future I imagine?
also, is this already designed?

Comment (mharr-cmpt) [reply src/components/home/MessageBox.js:32]: by designed i mean, written in code somewhere

File: src/components/home/CreateMessage.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ export default class CreateMessage extends Component {
     setModalVis(true);
     this.state.bounceValue.setValue(1);     // Start large
```
Comment (mharr-cmpt) [inline src/components/home/CreateMessage.js]: what's the concept of bounce?

File: src/components/home/Header.js (hunk 1 above)
Comment (tjmac21) [reply src/components/home/Header.js]: its an alert box. I did that for me just to remember what is happening at that point.

File: src/components/home/MessageBox.js (hunk 2 above)
Comment (tjmac21) [reply src/components/home/MessageBox.js:32]: No this is not made yet, I wanted your input in terms of how you think this object shoudl be structured and what other data should be included

File: src/components/home/Header.js (hunk 1 above)
Comment (mharr-cmpt) [reply src/components/home/Header.js]: ok i see... yeah i thoughtt he / did something fancier ;p

File: index.android.js [hunk 4]
```diff
@@ -1,54 +1,5 @@
...
-    flex: 1,
-    flexDirection: 'row',
-    justifyContent: 'center',
-    alignItems: 'center',
-    backgroundColor: '#F5FCFF',
-  },
-});
-
-AppRegistry.registerComponent('App2099', () => App2099);
+AppRegistry.registerComponent('App2099', () => App);
```
Comment (mharr-cmpt) [inline index.android.js:5]: what's this app name? App2099 => App, are they important at all?

File: src/components/home/MapContainer.js [hunk 5]
```diff
@@ -22,8 +43,7 @@ class MapTest extends Component {
                 zoomEnabled={true}
               >
               </MapView>
-              
-              <Text>FUUUCK</Text>
```
Comment (mharr-cmpt) [inline src/components/home/MapContainer.js:26]: why you take this out :D (do this to me)

File: index.android.js (hunk 4 above)
Comment (tjmac21) [reply index.android.js:5]: yes, we need to register the app, at the very beginning. 

App2099 on the right of the arrow made it seem specific to our app but instead App is more descriptive to just being the App app itself.

At the end its just a naming convention.

File: src/components/home/MapContainer.js (hunk 5 above)
Comment (tjmac21) [reply src/components/home/MapContainer.js:26]: 🤣

File: src/components/login/BackButton.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
```
Comment (mharr-cmpt) [inline src/components/login/BackButton.js:1]: does every button have to be a new .js file?

Comment (tjmac21) [reply src/components/login/BackButton.js:1]: No but it helps for portability and reusability

**Pull request merged**